h2/src/main/org/h2/command/dml/Select.java

```diff
@@ -1055,6 +1056,7 @@ public void init() {
         if (checkInit) {
             DbException.throwInternalError();
         }
+        Collections.sort(filters, TableFilter.ORDER_IN_FORM_COMPARATOR);
```

Comment:
what is a "form" ?

Reply:
I somehow typed “from” multiple times as “form”.

Reply:
lower IO latency from brain to right hand, than from brain to left hand ? :-)